game/templates/game/board.html

```
@@ -548,52 +520,96 @@ <h1>Checkora <span class="badge" id="modeBadge">PVP</span></h1>
+                        <span id="blackNameLabel">BLACK</span>
+                        <span id="blackYouTag"
+                            style="display: none; color: #f0c040; margin-left: 4px; font-size: 0.8em;">(YOU)</span>
+                    </span>
+                    <span class="time" id="blackTime">10:00</span>
                 </div>
             </div>
 
-            <button id="pauseBtn" class="btn btn-gold">Pause</button>
-            <button id="resignBtn" class="btn btn-danger" onclick="showResignModal()">Resign</button>
 
+            <button id="pauseBtn" class="btn btn-gold">Pause</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)